core/src/main/java/greencity/webcontroller/ManagementUserController.java

```diff
@@ -64,13 +55,19 @@ public String showForm(@RequestParam Long id, Model model) {
      * @return View template path {@link String}.
      * @author Vasyl Zhovnir
      */
-    @PostMapping("/update")
-    public String updateUser(@Valid @ModelAttribute("user") UserManagementDto userDto, BindingResult bindingResult) {
+    @PutMapping("")
+    @ResponseBody
+    public GenericResponseDto updateUser(@Valid @RequestBody UserManagementDto userDto, BindingResult bindingResult) {
         if (bindingResult.hasErrors()) {
-            return "core/management_user_update_form";
+            GenericResponseDto genericResponseDto = new GenericResponseDto();
+            for (FieldError fieldError : bindingResult.getFieldErrors()) {
+                genericResponseDto.getErrors().add(
+                    new FieldErrorDto(fieldError.getField(), fieldError.getDefaultMessage()));
+            }
+            return genericResponseDto;
         }
         userService.updateUser(userDto);
-        return "redirect:/management/users";
+        return GenericResponseDto.builder().errors(new ArrayList<>()).build();
```

Comment:
Is it possible to remove `.errors(new ArrayList<>())`. For example make it the builder responsibility to initialize the list as empty?

Reply:
Yes, done.